packages/sdk-client/src/sdk-middleware-user-agent/user-agent.ts

```diff
@@ -1,25 +1,19 @@
-import { default as createHttpUserAgent} from '../http-user-agent/create-user-agent'
-
+import { default as createHttpUserAgent } from '../http-user-agent/create-user-agent'
 import {
   Dispatch,
   Middleware,
   MiddlewareRequest,
   MiddlewareResponse,
+  UserAgentMiddlewareOptions
 } from '../types/sdk'
 
-type UserAgentMiddlewareOptions = {
-  libraryName?: string
-  libraryVersion?: string
-  contactUrl?: string
-  contactEmail?: string
-}
-
+const { version } = require('root-require')('package.json')
 export default function createUserAgentMiddleware(
   options: UserAgentMiddlewareOptions
 ): Middleware {
   const userAgent = createHttpUserAgent({
-    name: 'commercetools-js-sdk',
-    ...options,
+    name: `commercetools-sdk-javascript-v2/${version}`,
+    // ...options,
```

Comment:
is there a reason for commenting options?

Reply:
Yeah, 
The options are coming from the userAgent client middleware options
```
  libraryName: options.libraryName || '',
  libraryVersion: options.libraryVersion || '',
  contactUrl: options.contactUrl || '',
  contactEmail: options.contactEmail || '',
```
I figured that we don't need the customer to be passing in these data anymore, as the SDK will automatically generate the `libraryName` and `libraryVersion` by itsself.

Reply:
then i would removed the commented line

Reply:
Okay